[2023-03-08T17:50:21.841+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [queued]>
[2023-03-08T17:50:21.865+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [queued]>
[2023-03-08T17:50:21.867+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-08T17:50:21.869+0000] {taskinstance.py:1280} INFO - Starting attempt 9 of 9
[2023-03-08T17:50:21.870+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-08T17:50:21.908+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonVirtualenvOperator): chatgpt_prediction> on 2023-03-08 17:00:28.852949+00:00
[2023-03-08T17:50:21.921+0000] {standard_task_runner.py:55} INFO - Started process 3182 to run task
[2023-03-08T17:50:21.940+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bitcoin_second_webscrape', 'chatgpt_prediction', 'manual__2023-03-08T17:00:28.852949+00:00', '--job-id', '225', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp04cikvt0']
[2023-03-08T17:50:21.942+0000] {standard_task_runner.py:83} INFO - Job 225: Subtask chatgpt_prediction
[2023-03-08T17:50:22.109+0000] {task_command.py:388} INFO - Running <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]> on host cd114ba8c1f8
[2023-03-08T17:50:22.264+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bitcoin_second_webscrape
AIRFLOW_CTX_TASK_ID=chatgpt_prediction
AIRFLOW_CTX_EXECUTION_DATE=2023-03-08T17:00:28.852949+00:00
AIRFLOW_CTX_TRY_NUMBER=9
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-08T17:00:28.852949+00:00
[2023-03-08T17:50:22.273+0000] {process_utils.py:179} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvbbbbpdza --system-site-packages
[2023-03-08T17:50:22.301+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:50:23.463+0000] {process_utils.py:187} INFO - created virtual environment CPython3.7.16.final.0-64 in 585ms
[2023-03-08T17:50:23.466+0000] {process_utils.py:187} INFO -   creator CPython3Posix(dest=/tmp/venvbbbbpdza, clear=False, no_vcs_ignore=False, global=True)
[2023-03-08T17:50:23.468+0000] {process_utils.py:187} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-03-08T17:50:23.472+0000] {process_utils.py:187} INFO -     added seed packages: pip==23.0.1, setuptools==67.2.0, wheel==0.38.4
[2023-03-08T17:50:23.474+0000] {process_utils.py:187} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-03-08T17:50:23.492+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venvbbbbpdza/bin/pip install -r /tmp/venvbbbbpdza/requirements.txt
[2023-03-08T17:50:23.515+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:50:23.645+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2023-03-08T17:50:23.648+0000] {taskinstance.py:1077} INFO - Dependencies not met for <TaskInstance: bitcoin_second_webscrape.chatgpt_prediction manual__2023-03-08T17:00:28.852949+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2023-03-08T17:50:23.652+0000] {local_task_job.py:147} INFO - Task is not able to be run
[2023-03-08T17:50:26.105+0000] {process_utils.py:187} INFO - Collecting selenium
[2023-03-08T17:50:26.134+0000] {process_utils.py:187} INFO -   Using cached selenium-4.8.2-py3-none-any.whl (6.9 MB)
[2023-03-08T17:50:26.199+0000] {process_utils.py:187} INFO - Requirement already satisfied: pandas in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvbbbbpdza/requirements.txt (line 2)) (1.3.5)
[2023-03-08T17:50:26.284+0000] {process_utils.py:187} INFO - Collecting gcsfs
[2023-03-08T17:50:26.292+0000] {process_utils.py:187} INFO -   Using cached gcsfs-2023.1.0-py2.py3-none-any.whl (26 kB)
[2023-03-08T17:50:26.298+0000] {process_utils.py:187} INFO - Requirement already satisfied: fsspec in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvbbbbpdza/requirements.txt (line 4)) (2022.11.0)
[2023-03-08T17:50:26.300+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-storage in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvbbbbpdza/requirements.txt (line 5)) (2.7.0)
[2023-03-08T17:50:26.303+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvbbbbpdza/requirements.txt (line 6)) (2.34.4)
[2023-03-08T17:50:26.305+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvbbbbpdza/requirements.txt (line 7)) (2.16.0)
[2023-03-08T17:50:26.307+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-google in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvbbbbpdza/requirements.txt (line 8)) (8.8.0)
[2023-03-08T17:50:26.310+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***[google] in /home/***/.local/lib/python3.7/site-packages (from -r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.5.1)
[2023-03-08T17:50:26.391+0000] {process_utils.py:187} INFO - Collecting openai
[2023-03-08T17:50:26.397+0000] {process_utils.py:187} INFO -   Using cached openai-0.27.0-py3-none-any.whl (70 kB)
[2023-03-08T17:50:26.485+0000] {process_utils.py:187} INFO - Collecting trio~=0.17
[2023-03-08T17:50:26.491+0000] {process_utils.py:187} INFO -   Using cached trio-0.22.0-py3-none-any.whl (384 kB)
[2023-03-08T17:50:26.500+0000] {process_utils.py:187} INFO - Requirement already satisfied: urllib3[socks]~=1.26 in /home/***/.local/lib/python3.7/site-packages (from selenium->-r /tmp/venvbbbbpdza/requirements.txt (line 1)) (1.26.14)
[2023-03-08T17:50:26.570+0000] {process_utils.py:187} INFO - Collecting trio-websocket~=0.9
[2023-03-08T17:50:26.575+0000] {process_utils.py:187} INFO -   Using cached trio_websocket-0.9.2-py3-none-any.whl (16 kB)
[2023-03-08T17:50:26.580+0000] {process_utils.py:187} INFO - Requirement already satisfied: certifi>=2021.10.8 in /home/***/.local/lib/python3.7/site-packages (from selenium->-r /tmp/venvbbbbpdza/requirements.txt (line 1)) (2022.12.7)
[2023-03-08T17:50:26.602+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-dateutil>=2.7.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venvbbbbpdza/requirements.txt (line 2)) (2.8.2)
[2023-03-08T17:50:26.607+0000] {process_utils.py:187} INFO - Requirement already satisfied: numpy>=1.17.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venvbbbbpdza/requirements.txt (line 2)) (1.21.1)
[2023-03-08T17:50:26.612+0000] {process_utils.py:187} INFO - Requirement already satisfied: pytz>=2017.3 in /home/***/.local/lib/python3.7/site-packages (from pandas->-r /tmp/venvbbbbpdza/requirements.txt (line 2)) (2022.7.1)
[2023-03-08T17:50:26.747+0000] {process_utils.py:187} INFO - Collecting fsspec
[2023-03-08T17:50:26.754+0000] {process_utils.py:187} INFO -   Using cached fsspec-2023.1.0-py3-none-any.whl (143 kB)
[2023-03-08T17:50:26.759+0000] {process_utils.py:187} INFO - Requirement already satisfied: decorator>4.1.2 in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (5.1.1)
[2023-03-08T17:50:26.763+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiohttp!=4.0.0a0,!=4.0.0a1 in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (3.8.3)
[2023-03-08T17:50:26.768+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (2.28.2)
[2023-03-08T17:50:26.775+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth-oauthlib in /home/***/.local/lib/python3.7/site-packages (from gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (0.8.0)
[2023-03-08T17:50:26.853+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-resumable-media>=2.3.2 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvbbbbpdza/requirements.txt (line 5)) (2.4.0)
[2023-03-08T17:50:26.855+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-core<3.0dev,>=2.3.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvbbbbpdza/requirements.txt (line 5)) (2.3.2)
[2023-03-08T17:50:26.857+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-storage->-r /tmp/venvbbbbpdza/requirements.txt (line 5)) (2.8.2)
[2023-03-08T17:50:26.925+0000] {process_utils.py:187} INFO - Requirement already satisfied: protobuf<4.0.0dev,>=3.12.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvbbbbpdza/requirements.txt (line 6)) (3.20.0)
[2023-03-08T17:50:26.927+0000] {process_utils.py:187} INFO - Requirement already satisfied: proto-plus<2.0.0dev,>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvbbbbpdza/requirements.txt (line 6)) (1.19.6)
[2023-03-08T17:50:26.930+0000] {process_utils.py:187} INFO - Requirement already satisfied: packaging<22.0dev,>=14.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvbbbbpdza/requirements.txt (line 6)) (21.3)
[2023-03-08T17:50:26.933+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio<2.0dev,>=1.38.1 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigquery->-r /tmp/venvbbbbpdza/requirements.txt (line 6)) (1.51.1)
[2023-03-08T17:50:26.991+0000] {process_utils.py:187} INFO - Requirement already satisfied: cachetools<6.0,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvbbbbpdza/requirements.txt (line 7)) (4.2.2)
[2023-03-08T17:50:26.994+0000] {process_utils.py:187} INFO - Requirement already satisfied: rsa<5,>=3.1.4 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvbbbbpdza/requirements.txt (line 7)) (4.9)
[2023-03-08T17:50:26.996+0000] {process_utils.py:187} INFO - Requirement already satisfied: six>=1.9.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvbbbbpdza/requirements.txt (line 7)) (1.16.0)
[2023-03-08T17:50:26.999+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyasn1-modules>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from google-auth->-r /tmp/venvbbbbpdza/requirements.txt (line 7)) (0.2.8)
[2023-03-08T17:50:27.157+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-auth>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (4.1.5)
[2023-03-08T17:50:27.168+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dlp<2.0.0,>=0.11.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:50:27.173+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-bigquery>=1.2.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:50:27.176+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-ads>=15.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (18.0.0)
[2023-03-08T17:50:27.186+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery-datatransfer>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (3.7.0)
[2023-03-08T17:50:27.195+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-language<2.0.0,>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.3.2)
[2023-03-08T17:50:27.197+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataproc>=3.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (5.0.0)
[2023-03-08T17:50:27.199+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-bigquery>=6.1.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (6.2.0)
[2023-03-08T17:50:27.201+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-redis>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.9.0)
[2023-03-08T17:50:27.202+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-common-sql>=1.3.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.3.3)
[2023-03-08T17:50:27.204+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-translate<2.0.0,>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.7.2)
[2023-03-08T17:50:27.205+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-monitoring>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.11.0)
[2023-03-08T17:50:27.207+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-orchestration-***<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.4.1)
[2023-03-08T17:50:27.208+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-workflows<2.0.0,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.7.1)
[2023-03-08T17:50:27.210+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-spanner<2.0.0,>=1.10.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.19.3)
[2023-03-08T17:50:27.212+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-compute<2.0.0,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.7.0)
[2023-03-08T17:50:27.214+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigtable<2.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.7.3)
[2023-03-08T17:50:27.215+0000] {process_utils.py:187} INFO - Requirement already satisfied: asgiref>=3.5.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (3.6.0)
[2023-03-08T17:50:27.217+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-logging>=2.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (3.2.1)
[2023-03-08T17:50:27.220+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataplex>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.1.0)
[2023-03-08T17:50:27.222+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-tasks>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.10.1)
[2023-03-08T17:50:27.224+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-videointelligence<2.0.0,>=1.7.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.16.3)
[2023-03-08T17:50:27.226+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-datacatalog>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (3.9.0)
[2023-03-08T17:50:27.228+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-pubsub>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.13.5)
[2023-03-08T17:50:27.230+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-aiplatform<2.0.0,>=1.7.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.16.1)
[2023-03-08T17:50:27.232+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-automl>=2.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.8.0)
[2023-03-08T17:50:27.234+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-auth-httplib2>=0.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.1.0)
[2023-03-08T17:50:27.236+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-api-python-client<2.0.0,>=1.6.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.12.11)
[2023-03-08T17:50:27.238+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-secret-manager<2.0.0,>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:50:27.239+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-memcache>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.4.1)
[2023-03-08T17:50:27.241+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio-gcp>=0.2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.2.2)
[2023-03-08T17:50:27.243+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-build>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (3.9.0)
[2023-03-08T17:50:27.245+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataform>=0.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.2.0)
[2023-03-08T17:50:27.247+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-texttospeech<2.0.0,>=0.4.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.0.3)
[2023-03-08T17:50:27.249+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-kms>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.12.0)
[2023-03-08T17:50:27.252+0000] {process_utils.py:187} INFO - Requirement already satisfied: gcloud-aio-storage in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (8.0.0)
[2023-03-08T17:50:27.254+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-container<3.0.0,>=2.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.11.1)
[2023-03-08T17:50:27.255+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-os-login>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.7.1)
[2023-03-08T17:50:27.257+0000] {process_utils.py:187} INFO - Requirement already satisfied: PyOpenSSL in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (22.1.0)
[2023-03-08T17:50:27.258+0000] {process_utils.py:187} INFO - Requirement already satisfied: pandas-gbq in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.17.9)
[2023-03-08T17:50:27.260+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-speech<2.0.0,>=0.36.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.3.4)
[2023-03-08T17:50:27.263+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-dataproc-metastore<2.0.0,>=1.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.6.0)
[2023-03-08T17:50:27.264+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-vision<2.0.0,>=0.35.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.0.2)
[2023-03-08T17:50:27.266+0000] {process_utils.py:187} INFO - Requirement already satisfied: json-merge-patch>=0.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.2)
[2023-03-08T17:50:27.268+0000] {process_utils.py:187} INFO - Requirement already satisfied: looker-sdk>=22.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (22.20.0)
[2023-03-08T17:50:27.270+0000] {process_utils.py:187} INFO - Requirement already satisfied: httpx in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.23.3)
[2023-03-08T17:50:32.936+0000] {process_utils.py:187} INFO - Requirement already satisfied: markdown-it-py>=2.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.1.0)
[2023-03-08T17:50:32.938+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-daemon>=2.2.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.3.2)
[2023-03-08T17:50:32.940+0000] {process_utils.py:187} INFO - Requirement already satisfied: graphviz>=0.12 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.20.1)
[2023-03-08T17:50:32.942+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-oneofschema>=2.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.0.1)
[2023-03-08T17:50:32.946+0000] {process_utils.py:187} INFO - Requirement already satisfied: pathspec~=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:50:32.952+0000] {process_utils.py:187} INFO - Requirement already satisfied: importlib-metadata>=1.7 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (4.13.0)
[2023-03-08T17:50:32.954+0000] {process_utils.py:187} INFO - Requirement already satisfied: connexion[flask]>=2.10.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.14.1)
[2023-03-08T17:50:32.955+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-sqlite in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.3.1)
[2023-03-08T17:50:32.957+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-http in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (4.1.1)
[2023-03-08T17:50:32.959+0000] {process_utils.py:187} INFO - Requirement already satisfied: blinker in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.5)
[2023-03-08T17:50:32.962+0000] {process_utils.py:187} INFO - Requirement already satisfied: lazy-object-proxy in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.9.0)
[2023-03-08T17:50:32.964+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy<2.0,>=1.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.4.46)
[2023-03-08T17:50:32.967+0000] {process_utils.py:187} INFO - Requirement already satisfied: colorlog<5.0,>=4.0.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (4.8.0)
[2023-03-08T17:50:32.969+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-slugify>=5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (7.0.0)
[2023-03-08T17:50:32.971+0000] {process_utils.py:187} INFO - Requirement already satisfied: unicodecsv>=0.14.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.14.1)
[2023-03-08T17:50:32.972+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-jsonfield>=1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.0.1.post0)
[2023-03-08T17:50:32.974+0000] {process_utils.py:187} INFO - Requirement already satisfied: typing-extensions>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (4.4.0)
[2023-03-08T17:50:32.975+0000] {process_utils.py:187} INFO - Requirement already satisfied: argcomplete>=1.10 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:50:32.976+0000] {process_utils.py:187} INFO - Requirement already satisfied: markupsafe>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:50:32.978+0000] {process_utils.py:187} INFO - Requirement already satisfied: deprecated>=1.2.13 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.2.13)
[2023-03-08T17:50:32.979+0000] {process_utils.py:187} INFO - Requirement already satisfied: pluggy>=1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.0.0)
[2023-03-08T17:50:32.981+0000] {process_utils.py:187} INFO - Requirement already satisfied: setproctitle>=1.1.8 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.3.2)
[2023-03-08T17:50:32.984+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-session>=0.4.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.4.0)
[2023-03-08T17:50:32.986+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-wtf>=0.15 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.1.1)
[2023-03-08T17:50:32.988+0000] {process_utils.py:187} INFO - Requirement already satisfied: mdit-py-plugins>=0.3.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.3.3)
[2023-03-08T17:50:32.990+0000] {process_utils.py:187} INFO - Requirement already satisfied: croniter>=0.3.17 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.3.8)
[2023-03-08T17:50:32.991+0000] {process_utils.py:187} INFO - Requirement already satisfied: alembic<2.0,>=1.6.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.9.2)
[2023-03-08T17:50:32.994+0000] {process_utils.py:187} INFO - Requirement already satisfied: pygments>=2.0.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.14.0)
[2023-03-08T17:50:32.995+0000] {process_utils.py:187} INFO - Requirement already satisfied: cached-property>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.5.2)
[2023-03-08T17:50:32.997+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-imap in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.1.1)
[2023-03-08T17:50:32.998+0000] {process_utils.py:187} INFO - Requirement already satisfied: gunicorn>=20.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (20.1.0)
[2023-03-08T17:50:32.999+0000] {process_utils.py:187} INFO - Requirement already satisfied: cattrs>=22.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (22.2.0)
[2023-03-08T17:50:33.001+0000] {process_utils.py:187} INFO - Requirement already satisfied: cron-descriptor>=1.2.24 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.2.32)
[2023-03-08T17:50:33.002+0000] {process_utils.py:187} INFO - Requirement already satisfied: jsonschema>=3.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (4.17.3)
[2023-03-08T17:50:33.004+0000] {process_utils.py:187} INFO - Requirement already satisfied: pendulum>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:50:33.005+0000] {process_utils.py:187} INFO - Requirement already satisfied: linkify-it-py>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:50:33.007+0000] {process_utils.py:187} INFO - Requirement already satisfied: apache-***-providers-ftp in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.3.0)
[2023-03-08T17:50:33.009+0000] {process_utils.py:187} INFO - Requirement already satisfied: lockfile>=0.12.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.12.2)
[2023-03-08T17:50:33.011+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyjwt>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.6.0)
[2023-03-08T17:50:33.014+0000] {process_utils.py:187} INFO - Requirement already satisfied: dill>=0.2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.3.1.1)
[2023-03-08T17:50:33.015+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-login>=0.6.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.6.2)
[2023-03-08T17:50:33.017+0000] {process_utils.py:187} INFO - Requirement already satisfied: itsdangerous>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.1.2)
[2023-03-08T17:50:33.018+0000] {process_utils.py:187} INFO - Requirement already satisfied: psutil>=4.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (5.9.4)
[2023-03-08T17:50:33.020+0000] {process_utils.py:187} INFO - Requirement already satisfied: tabulate>=0.7.5 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:50:33.022+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-caching>=1.5.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.0.2)
[2023-03-08T17:50:33.023+0000] {process_utils.py:187} INFO - Requirement already satisfied: markdown>=3.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.4.1)
[2023-03-08T17:50:33.031+0000] {process_utils.py:187} INFO - Requirement already satisfied: cryptography>=0.9.3 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (38.0.4)
[2023-03-08T17:50:33.035+0000] {process_utils.py:187} INFO - Requirement already satisfied: jinja2>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.1.2)
[2023-03-08T17:50:33.037+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask-appbuilder==4.1.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (4.1.4)
[2023-03-08T17:50:33.038+0000] {process_utils.py:187} INFO - Requirement already satisfied: attrs>=22.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (22.2.0)
[2023-03-08T17:50:33.040+0000] {process_utils.py:187} INFO - Requirement already satisfied: rich>=12.4.4 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (13.1.0)
[2023-03-08T17:50:33.043+0000] {process_utils.py:187} INFO - Requirement already satisfied: werkzeug>=2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.2.2)
[2023-03-08T17:50:33.047+0000] {process_utils.py:187} INFO - Requirement already satisfied: python-nvd3>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.15.0)
[2023-03-08T17:50:33.050+0000] {process_utils.py:187} INFO - Requirement already satisfied: tenacity>=6.2.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (8.1.0)
[2023-03-08T17:50:33.053+0000] {process_utils.py:187} INFO - Requirement already satisfied: termcolor>=1.1.0 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.2.0)
[2023-03-08T17:50:33.057+0000] {process_utils.py:187} INFO - Requirement already satisfied: importlib-resources>=5.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (5.10.2)
[2023-03-08T17:50:33.059+0000] {process_utils.py:187} INFO - Requirement already satisfied: flask<2.3,>=2.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.2.2)
[2023-03-08T17:50:33.062+0000] {process_utils.py:187} INFO - Requirement already satisfied: configupdater>=3.1.1 in /home/***/.local/lib/python3.7/site-packages (from apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.1.1)
[2023-03-08T17:50:33.085+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-Babel<3,>=1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.0.0)
[2023-03-08T17:50:33.088+0000] {process_utils.py:187} INFO - Requirement already satisfied: apispec[yaml]<4,>=3.3 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.3.2)
[2023-03-08T17:50:33.090+0000] {process_utils.py:187} INFO - Requirement already satisfied: colorama<1,>=0.3.9 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.4.6)
[2023-03-08T17:50:33.092+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-sqlalchemy<0.27.0,>=0.22.0 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.26.1)
[2023-03-08T17:50:33.095+0000] {process_utils.py:187} INFO - Requirement already satisfied: click<9,>=8 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (8.1.3)
[2023-03-08T17:50:33.104+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow<4,>=3 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.19.0)
[2023-03-08T17:50:33.107+0000] {process_utils.py:187} INFO - Requirement already satisfied: email-validator<2,>=1.0.5 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.3.0)
[2023-03-08T17:50:33.118+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlalchemy-utils<1,>=0.32.21 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.39.0)
[2023-03-08T17:50:33.120+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-JWT-Extended<5.0.0,>=4.0.0 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (4.4.4)
[2023-03-08T17:50:33.123+0000] {process_utils.py:187} INFO - Requirement already satisfied: Flask-SQLAlchemy<3,>=2.4 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.5.1)
[2023-03-08T17:50:33.125+0000] {process_utils.py:187} INFO - Requirement already satisfied: WTForms<4 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.0.1)
[2023-03-08T17:50:33.158+0000] {process_utils.py:187} INFO - Requirement already satisfied: prison<1.0.0,>=0.2.1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.2.1)
[2023-03-08T17:50:33.161+0000] {process_utils.py:187} INFO - Requirement already satisfied: marshmallow-enum<2,>=1.5.1 in /home/***/.local/lib/python3.7/site-packages (from flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.5.1)
[2023-03-08T17:50:33.378+0000] {process_utils.py:187} INFO - Collecting tqdm
[2023-03-08T17:50:33.382+0000] {process_utils.py:187} INFO -   Using cached tqdm-4.65.0-py3-none-any.whl (77 kB)
[2023-03-08T17:50:33.424+0000] {process_utils.py:187} INFO - Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (4.0.2)
[2023-03-08T17:50:33.426+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiosignal>=1.1.2 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (1.3.1)
[2023-03-08T17:50:33.429+0000] {process_utils.py:187} INFO - Requirement already satisfied: frozenlist>=1.1.1 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (1.3.3)
[2023-03-08T17:50:33.431+0000] {process_utils.py:187} INFO - Requirement already satisfied: asynctest==0.13.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (0.13.0)
[2023-03-08T17:50:33.433+0000] {process_utils.py:187} INFO - Requirement already satisfied: charset-normalizer<3.0,>=2.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (2.1.1)
[2023-03-08T17:50:33.436+0000] {process_utils.py:187} INFO - Requirement already satisfied: multidict<7.0,>=4.5 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (6.0.4)
[2023-03-08T17:50:33.438+0000] {process_utils.py:187} INFO - Requirement already satisfied: yarl<2.0,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from aiohttp!=4.0.0a0,!=4.0.0a1->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (1.8.2)
[2023-03-08T17:50:33.474+0000] {process_utils.py:187} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.7/site-packages (from alembic<2.0,>=1.6.3->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.2.4)
[2023-03-08T17:50:33.512+0000] {process_utils.py:187} INFO - Requirement already satisfied: sqlparse>=0.4.2 in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-common-sql>=1.3.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.4.3)
[2023-03-08T17:50:33.651+0000] {process_utils.py:187} INFO - Requirement already satisfied: exceptiongroup in /home/***/.local/lib/python3.7/site-packages (from cattrs>=22.1.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.1.0)
[2023-03-08T17:50:33.754+0000] {process_utils.py:187} INFO - Requirement already satisfied: clickclick<21,>=1.2 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (20.10.2)
[2023-03-08T17:50:33.756+0000] {process_utils.py:187} INFO - Requirement already satisfied: PyYAML<7,>=5.1 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (6.0)
[2023-03-08T17:50:33.760+0000] {process_utils.py:187} INFO - Requirement already satisfied: inflection<0.6,>=0.3.1 in /home/***/.local/lib/python3.7/site-packages (from connexion[flask]>=2.10.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.5.1)
[2023-03-08T17:50:33.831+0000] {process_utils.py:187} INFO - Requirement already satisfied: cffi>=1.12 in /home/***/.local/lib/python3.7/site-packages (from cryptography>=0.9.3->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.15.1)
[2023-03-08T17:50:33.882+0000] {process_utils.py:187} INFO - Requirement already satisfied: wrapt<2,>=1.10 in /home/***/.local/lib/python3.7/site-packages (from deprecated>=1.2.13->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.14.1)
[2023-03-08T17:50:33.931+0000] {process_utils.py:187} INFO - Requirement already satisfied: cachelib<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from flask-caching>=1.5.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.9.0)
[2023-03-08T17:50:34.002+0000] {process_utils.py:187} INFO - Requirement already satisfied: future<0.19.0,>=0.17.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.18.3)
[2023-03-08T17:50:34.007+0000] {process_utils.py:187} INFO - Requirement already satisfied: chardet<4.1,>=2.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (4.0.0)
[2023-03-08T17:50:34.008+0000] {process_utils.py:187} INFO - Requirement already satisfied: backoff<3.0.0,>=1.0.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-auth>=4.0.0->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.10.0)
[2023-03-08T17:50:34.057+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpcio-status<2.0.0,>=1.38.1 in /home/***/.local/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.48.2)
[2023-03-08T17:50:34.059+0000] {process_utils.py:187} INFO - Requirement already satisfied: setuptools>=40.3.0 in /tmp/venvbbbbpdza/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (67.2.0)
[2023-03-08T17:50:34.061+0000] {process_utils.py:187} INFO - Requirement already satisfied: googleapis-common-protos<2.0.0,>=1.5.8 in /home/***/.local/lib/python3.7/site-packages (from google-ads>=15.1.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.56.4)
[2023-03-08T17:50:34.147+0000] {process_utils.py:187} INFO - Requirement already satisfied: uritemplate<4dev,>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from google-api-python-client<2.0.0,>=1.6.0->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (3.0.1)
[2023-03-08T17:50:34.150+0000] {process_utils.py:187} INFO - Requirement already satisfied: httplib2<1dev,>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from google-api-python-client<2.0.0,>=1.6.0->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.20.4)
[2023-03-08T17:50:34.187+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests-oauthlib>=0.7.0 in /home/***/.local/lib/python3.7/site-packages (from google-auth-oauthlib->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (1.3.1)
[2023-03-08T17:50:34.350+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-resource-manager<3.0.0dev,>=1.3.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-aiplatform<2.0.0,>=1.7.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.6.0)
[2023-03-08T17:50:34.431+0000] {process_utils.py:187} INFO - Requirement already satisfied: grpc-google-iam-v1<0.13dev,>=0.12.3 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-bigtable<2.0.0,>=1.0.0->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.12.4)
[2023-03-08T17:50:34.796+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-appengine-logging<2.0.0dev,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-logging>=2.1.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.1.3)
[2023-03-08T17:50:34.806+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-audit-log<1.0.0dev,>=0.1.0 in /home/***/.local/lib/python3.7/site-packages (from google-cloud-logging>=2.1.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.2.4)
[2023-03-08T17:50:35.548+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-crc32c<2.0dev,>=1.0 in /home/***/.local/lib/python3.7/site-packages (from google-resumable-media>=2.3.2->google-cloud-storage->-r /tmp/venvbbbbpdza/requirements.txt (line 5)) (1.5.0)
[2023-03-08T17:50:35.931+0000] {process_utils.py:187} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.7/site-packages (from importlib-metadata>=1.7->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (3.11.0)
[2023-03-08T17:50:36.228+0000] {process_utils.py:187} INFO - Requirement already satisfied: pkgutil-resolve-name>=1.3.10 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.2.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.3.10)
[2023-03-08T17:50:36.231+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0 in /home/***/.local/lib/python3.7/site-packages (from jsonschema>=3.2.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.19.3)
[2023-03-08T17:50:36.285+0000] {process_utils.py:187} INFO - Requirement already satisfied: uc-micro-py in /home/***/.local/lib/python3.7/site-packages (from linkify-it-py>=2.0.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.0.1)
[2023-03-08T17:50:36.484+0000] {process_utils.py:187} INFO - Requirement already satisfied: mdurl~=0.1 in /home/***/.local/lib/python3.7/site-packages (from markdown-it-py>=2.1.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.1.2)
[2023-03-08T17:50:36.706+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/***/.local/lib/python3.7/site-packages (from packaging<22.0dev,>=14.3->google-cloud-bigquery->-r /tmp/venvbbbbpdza/requirements.txt (line 6)) (3.0.9)
[2023-03-08T17:50:36.790+0000] {process_utils.py:187} INFO - Requirement already satisfied: pytzdata>=2020.1 in /home/***/.local/lib/python3.7/site-packages (from pendulum>=2.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2020.1)
[2023-03-08T17:50:37.003+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /home/***/.local/lib/python3.7/site-packages (from pyasn1-modules>=0.2.1->google-auth->-r /tmp/venvbbbbpdza/requirements.txt (line 7)) (0.4.8)
[2023-03-08T17:50:37.193+0000] {process_utils.py:187} INFO - Requirement already satisfied: docutils in /home/***/.local/lib/python3.7/site-packages (from python-daemon>=2.2.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.19)
[2023-03-08T17:50:37.307+0000] {process_utils.py:187} INFO - Requirement already satisfied: text-unidecode>=1.3 in /home/***/.local/lib/python3.7/site-packages (from python-slugify>=5.0->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (1.3)
[2023-03-08T17:50:37.391+0000] {process_utils.py:187} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.7/site-packages (from requests->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (3.4)
[2023-03-08T17:50:37.451+0000] {process_utils.py:187} INFO - Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/***/.local/lib/python3.7/site-packages (from rich>=12.4.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.9.1)
[2023-03-08T17:50:37.722+0000] {process_utils.py:187} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy<2.0,>=1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.0.1)
[2023-03-08T17:50:37.996+0000] {process_utils.py:187} INFO - Requirement already satisfied: pyarrow>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy-bigquery>=1.2.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (9.0.0)
[2023-03-08T17:50:38.006+0000] {process_utils.py:187} INFO - Requirement already satisfied: google-cloud-bigquery-storage<3.0.0dev,>=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from sqlalchemy-bigquery>=1.2.1->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (2.14.1)
[2023-03-08T17:50:38.988+0000] {process_utils.py:187} INFO - Collecting outcome
[2023-03-08T17:50:39.001+0000] {process_utils.py:187} INFO -   Using cached outcome-1.2.0-py2.py3-none-any.whl (9.7 kB)
[2023-03-08T17:50:39.004+0000] {process_utils.py:187} INFO - Requirement already satisfied: sniffio in /home/***/.local/lib/python3.7/site-packages (from trio~=0.17->selenium->-r /tmp/venvbbbbpdza/requirements.txt (line 1)) (1.3.0)
[2023-03-08T17:50:39.006+0000] {process_utils.py:187} INFO - Requirement already satisfied: sortedcontainers in /home/***/.local/lib/python3.7/site-packages (from trio~=0.17->selenium->-r /tmp/venvbbbbpdza/requirements.txt (line 1)) (2.4.0)
[2023-03-08T17:50:39.071+0000] {process_utils.py:187} INFO - Collecting async-generator>=1.9
[2023-03-08T17:50:39.078+0000] {process_utils.py:187} INFO -   Using cached async_generator-1.10-py3-none-any.whl (18 kB)
[2023-03-08T17:50:39.203+0000] {process_utils.py:187} INFO - Collecting wsproto>=0.14
[2023-03-08T17:50:39.204+0000] {process_utils.py:187} INFO -   Using cached wsproto-1.2.0-py3-none-any.whl (24 kB)
[2023-03-08T17:50:39.641+0000] {process_utils.py:187} INFO - Collecting PySocks!=1.5.7,<2.0,>=1.5.6
[2023-03-08T17:50:39.649+0000] {process_utils.py:187} INFO -   Using cached PySocks-1.7.1-py3-none-any.whl (16 kB)
[2023-03-08T17:50:39.846+0000] {process_utils.py:187} INFO - Requirement already satisfied: requests-toolbelt in /home/***/.local/lib/python3.7/site-packages (from apache-***-providers-http->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (0.10.1)
[2023-03-08T17:50:40.263+0000] {process_utils.py:187} INFO - Requirement already satisfied: aiofiles<23.0.0,>=0.6.0 in /home/***/.local/lib/python3.7/site-packages (from gcloud-aio-storage->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (22.1.0)
[2023-03-08T17:50:40.368+0000] {process_utils.py:187} INFO - Requirement already satisfied: httpcore<0.17.0,>=0.15.0 in /home/***/.local/lib/python3.7/site-packages (from httpx->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.16.3)
[2023-03-08T17:50:40.371+0000] {process_utils.py:187} INFO - Requirement already satisfied: rfc3986[idna2008]<2,>=1.3 in /home/***/.local/lib/python3.7/site-packages (from httpx->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:50:40.681+0000] {process_utils.py:187} INFO - Requirement already satisfied: pydata-google-auth in /home/***/.local/lib/python3.7/site-packages (from pandas-gbq->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.5.0)
[2023-03-08T17:50:40.685+0000] {process_utils.py:187} INFO - Requirement already satisfied: db-dtypes<2.0.0,>=0.3.1 in /home/***/.local/lib/python3.7/site-packages (from pandas-gbq->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (1.0.5)
[2023-03-08T17:50:41.576+0000] {process_utils.py:187} INFO - Requirement already satisfied: pycparser in /home/***/.local/lib/python3.7/site-packages (from cffi>=1.12->cryptography>=0.9.3->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.21)
[2023-03-08T17:50:42.178+0000] {process_utils.py:187} INFO - Requirement already satisfied: dnspython>=1.15.0 in /home/***/.local/lib/python3.7/site-packages (from email-validator<2,>=1.0.5->flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.3.0)
[2023-03-08T17:50:42.303+0000] {process_utils.py:187} INFO - Requirement already satisfied: Babel>=2.3 in /home/***/.local/lib/python3.7/site-packages (from Flask-Babel<3,>=1->flask-appbuilder==4.1.4->apache-***[google]->-r /tmp/venvbbbbpdza/requirements.txt (line 9)) (2.11.0)
[2023-03-08T17:50:42.725+0000] {process_utils.py:187} INFO - WARNING: google-api-core 2.8.2 does not provide the extra 'grpcgcp'
[2023-03-08T17:50:44.348+0000] {process_utils.py:187} INFO - Requirement already satisfied: h11<0.15,>=0.13 in /home/***/.local/lib/python3.7/site-packages (from httpcore<0.17.0,>=0.15.0->httpx->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (0.14.0)
[2023-03-08T17:50:44.353+0000] {process_utils.py:187} INFO - Requirement already satisfied: anyio<5.0,>=3.0 in /home/***/.local/lib/python3.7/site-packages (from httpcore<0.17.0,>=0.15.0->httpx->apache-***-providers-google->-r /tmp/venvbbbbpdza/requirements.txt (line 8)) (3.6.2)
[2023-03-08T17:50:46.058+0000] {process_utils.py:187} INFO - Requirement already satisfied: oauthlib>=3.0.0 in /home/***/.local/lib/python3.7/site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib->gcsfs->-r /tmp/venvbbbbpdza/requirements.txt (line 3)) (3.2.2)
[2023-03-08T17:50:51.281+0000] {process_utils.py:187} INFO - Installing collected packages: tqdm, PySocks, outcome, fsspec, async-generator, wsproto, trio, trio-websocket, openai, selenium, gcsfs
[2023-03-08T17:50:51.383+0000] {process_utils.py:187} INFO -   Attempting uninstall: fsspec
[2023-03-08T17:50:51.387+0000] {process_utils.py:187} INFO -     Found existing installation: fsspec 2022.11.0
[2023-03-08T17:50:51.388+0000] {process_utils.py:187} INFO -     Not uninstalling fsspec at /home/***/.local/lib/python3.7/site-packages, outside environment /tmp/venvbbbbpdza
[2023-03-08T17:50:51.390+0000] {process_utils.py:187} INFO -     Can't uninstall 'fsspec'. No files were found to uninstall.
[2023-03-08T17:50:53.143+0000] {process_utils.py:187} INFO - Successfully installed PySocks-1.7.1 async-generator-1.10 fsspec-2023.1.0 gcsfs-2023.1.0 openai-0.27.0 outcome-1.2.0 selenium-4.8.2 tqdm-4.65.0 trio-0.22.0 trio-websocket-0.9.2 wsproto-1.2.0
[2023-03-08T17:50:53.578+0000] {process_utils.py:179} INFO - Executing cmd: /tmp/venvbbbbpdza/bin/python /tmp/venvbbbbpdza/script.py /tmp/venvbbbbpdza/script.in /tmp/venvbbbbpdza/script.out /tmp/venvbbbbpdza/string_args.txt
[2023-03-08T17:50:53.599+0000] {process_utils.py:183} INFO - Output:
[2023-03-08T17:50:55.716+0000] {process_utils.py:187} INFO - /home/***/.local/lib/python3.7/site-packages/***/models/base.py:49 MovedIn20Warning: Deprecated API features detected! These feature(s) are not compatible with SQLAlchemy 2.0. To prevent incompatible upgrades prior to updating applications, ensure requirements files are pinned to "sqlalchemy<2.0". Set environment variable SQLALCHEMY_WARN_20=1 to show all deprecation warnings.  Set environment variable SQLALCHEMY_SILENCE_UBER_WARNING=1 to silence this message. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
[2023-03-08T17:50:57.966+0000] {process_utils.py:187} INFO - Traceback (most recent call last):
[2023-03-08T17:50:57.968+0000] {process_utils.py:187} INFO -   File "/tmp/venvbbbbpdza/script.py", line 64, in <module>
[2023-03-08T17:50:57.972+0000] {process_utils.py:187} INFO -     res = chat_gpt_prediction(*arg_dict["args"], **arg_dict["kwargs"])
[2023-03-08T17:50:57.975+0000] {process_utils.py:187} INFO -   File "/tmp/venvbbbbpdza/script.py", line 43, in chat_gpt_prediction
[2023-03-08T17:50:57.977+0000] {process_utils.py:187} INFO -     openai.api_key = config['OPENAI_API_KEY']
[2023-03-08T17:50:57.979+0000] {process_utils.py:187} INFO - KeyError: 'OPENAI_API_KEY'
[2023-03-08T17:50:58.435+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 356, in execute
    return super().execute(context=serializable_context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 553, in execute_callable
    return self._execute_python_callable_in_subprocess(python_path, tmp_path)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 418, in _execute_python_callable_in_subprocess
    os.fspath(string_args_path),
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 168, in execute_in_subprocess
    execute_in_subprocess_with_kwargs(cmd, cwd=cwd)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/process_utils.py", line 191, in execute_in_subprocess_with_kwargs
    raise subprocess.CalledProcessError(exit_code, cmd)
subprocess.CalledProcessError: Command '['/tmp/venvbbbbpdza/bin/python', '/tmp/venvbbbbpdza/script.py', '/tmp/venvbbbbpdza/script.in', '/tmp/venvbbbbpdza/script.out', '/tmp/venvbbbbpdza/string_args.txt']' returned non-zero exit status 1.
[2023-03-08T17:50:58.453+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=bitcoin_second_webscrape, task_id=chatgpt_prediction, execution_date=20230308T170028, start_date=20230308T175021, end_date=20230308T175058
[2023-03-08T17:50:58.479+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 225 for task chatgpt_prediction (Command '['/tmp/venvbbbbpdza/bin/python', '/tmp/venvbbbbpdza/script.py', '/tmp/venvbbbbpdza/script.in', '/tmp/venvbbbbpdza/script.out', '/tmp/venvbbbbpdza/string_args.txt']' returned non-zero exit status 1.; 3182)
[2023-03-08T17:50:58.526+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-08T17:50:58.552+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
